### DISCUSSION TITLE
What is the right way to put the page or App in 'Dark' mode?

### DISCUSSION THREAD

**Discussion by HDAlbarenque** [reflex-dev/reflex#2804] [Q&A]
Following the example of the official documentation, change "light" by "dark", leaving the code as follows:
`app = rx.App(
    theme=rx.theme(
        appearance="dark", has_background=True, radius="large", accent_color="teal"
    )
)`
But the page continues to be seen in "Light" mode.
I am looking for documentation, but I can't find anything else.
What am I wrongly interpreting?
reflex = 0.4.3

**Comment by masenf** [comment]
The color mode is "sticky". If you load the app with `appearance="light"` then it will stay in light mode, even if the default changes. You can try to view the app in an incognito tab, or clear the localstorage in the browser for the app and it should revert back to the mode specified in the theme.

This comes up often enough though, that i think we'll consider NOT making the default sticky, even though it might make the implementation a little more complicated internally.
